server/lib/whispers/whisper-service.ts

```diff
@@ -10,6 +13,8 @@ import {
   WhisperServiceErrorCode,
   WhisperSessionInitEvent,
 } from '../../../common/whispers'
+import { getChannelInfos } from '../chat/chat-models'
+import ChatService, { toBasicChannelInfo } from '../chat/chat-service'
```

Comment:
imo you should move `toBasicChannelInfo` into a different file, I find it weird that this service would ever need to import the chat service